src/tests/unit/studentFormData.test.js

```diff
@@ -0,0 +1,43 @@
+
+const filterFormData = (testData, cohortId) => testData.filter(app => app.id === cohortId)
```

Comment:
It's best that you import the function you're trying to test not rewrite it. One of the good things about unit tests is that it documents how the function works so if someone changes a critical piece of the functionality the test will fail and they will need to think about whether they are changing something they shouldn't be